package.json

```
@@ -9,6 +9,7 @@
     "build": "tsc && vite build",
     "check": "tsc --incremental --noEmit && rm -f tsconfig.tsbuildinfo",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint:fix": "eslint . --ext ts,tsx --fix",
```

```suggestion
    "lint:fix": "eslint --cache . --ext ts,tsx --fix",
```
I think it's can cache and optimization it